presto-druid/src/main/java/io/prestosql/plugin/druid/DruidConfig.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.druid;
+
+import io.airlift.configuration.Config;
+
+public class DruidConfig
+{
+    private String sqlTimeZone = "UTC";
+    private boolean useApproximateCountDistinct = true;
+    private boolean useApproximateTopN = true;
+
+    public String getSqlTimeZone()
+    {
+        return this.sqlTimeZone;
+    }
+
+    @Config("druid.sql-time-zone")
```

Comment:
What are semantics of this setting? Why do we need it?
(we don't need this in other JDBC-based connectors)

Reply:
Druid allows setting this a connection property https://druid.apache.org/docs/latest/querying/sql#connection-context. 
So exposed the setting the connector as well.

Reply:
Is this similar to session zone in Presto?
If so, exposing this as configuration is not correct, as changing the setting would affect correctness of Druid queries Presto generates.

Please explain semantics of this thing on druid side.

Reply:
It sets the time zone for the connection, which affects how time functions and timestamp literals behave.

Reply:
Let me know if you would still like me to remove the config.

Reply:
The config changes the plugin behavior, but is independent from the session zone. So i guess there is actually one config value yielding correct results -- UTC? same as JVM zone? same as session zone? i don't know... I would suggest that you add tests similar to `io.prestosql.plugin.postgresql.TestPostgreSqlTypeMapping#testTimestamp`

Reply:
Removed the druid.sql-time-zone config for now.